{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-4692",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/171829",
                    "name": "ibm-gde-cve20194692-info-disc (171829)",
                    "refsource": "XF",
                    "tags": []
                },
                {
                    "url": "https://www.ibm.com/support/pages/node/6320817",
                    "name": "https://www.ibm.com/support/pages/node/6320817",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "IBM Security Guardium Data Encryption (GDE) 3.0.0.2 discloses sensitive information to unauthorized users. The information can be used to mount further attacks on the system.  IBM X-Force ID:  171829."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-08-26T19:15Z",
    "lastModifiedDate": "2020-08-26T19:15Z"
}